modules/301-prometheus-metrics-adapter/images/prometheus-reverse-proxy/app/server/logger.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"log"
+	"os"
+)
+
+var (
+	infLog = log.New(os.Stdout, "I", log.LstdFlags)
```

Comment:
I often use `kubectl logs blah-blah | grep -i error` for check troubles in log.

I think, better use prefix `ERROR` and `INFO`.